src/app/pages/dashboard/widgets/apps/widget-app/widget-app.component.html

```diff
@@ -0,0 +1,171 @@
+              {{ app.name }}
+            </h3>
+            <div *ixWithLoadingState="application() as app" class="version">
+              v{{ app.chart_metadata?.appVersion }}
+            </div>
+          </div>
+          <div class="app-status">
+            <ix-app-status-cell
+              *ixWithLoadingState="application() as app"
+              [app]="app"
+              [job]="appJob()"
+              [showIcon]="true"
+            ></ix-app-status-cell>
+            <ix-app-update-cell
+              *ixWithLoadingState="application() as app"
+              [app]="app"
+              [showIcon]="true"
+            ></ix-app-update-cell>
+          </div>
+        </div>
+      </div>
+
+      <div class="app-stats">
+        <div class="app-stats-row">
+          <div class="slot square">
+            <div>
+              <h3 *ixWithLoadingState="stats() as stats">
+                <span>{{ stats.cpu.toFixed(0) }}</span>
+                <small>%</small>
+              </h3>
+              <div><strong>{{ 'CPU Usage' | translate }}</strong></div>
+            </div>
+          </div>
+          <div class="slot rectangle">
+            <div class="chart-info">
+              <h4>{{ 'Network I/O' | translate }}</h4>
+              <div class="in-out">
+                <div class="in-out-row">
+                  <span>{{ 'In' | translate }}:</span>
+                  <span *ixWithLoadingState="stats() as stats">
+                    {{ stats.network.incoming | ixFileSize: { baseUnit: 'b', base: 10 } }}/s
+                  </span>
+                </div>
+                <div class="in-out-row">
+                  <span>{{ 'Out' | translate }}:</span>
+                  <span *ixWithLoadingState="stats() as stats">
+                    {{ stats.network.outgoing | ixFileSize: { baseUnit: 'b', base: 10 } }}/s
+                  </span>
+                </div>
+              </div>
+              <small>{{ '1m Average' | translate }}</small>
+            </div>
+            <div class="chart-graph">
+              <ng-container *ngIf="(stats() | async).isLoading; then chartLoader; else networkChart"></ng-container>
+            </div>
+          </div>
+        </div>
+        <div class="app-stats-row">
+          <div class="slot square">
+            <div>
+              <h3>
+                <ng-container *ixWithLoadingState="stats() as stats">
+                  <ng-container *appLet="splitMemory(stats.memory | ixFileSize) as memory">
+                    <span>{{ memory[0].toFixed(0) }}</span>
+                    <small>{{ memory[1] }}</small>
+                  </ng-container>
+                </ng-container>
+              </h3>
+              <div>
+                <strong>{{ 'Memory Usage' | translate }}</strong>
+              </div>
+            </div>
+          </div>
+          <div class="slot rectangle">
+            <div class="chart-info">
+              <h4>{{ 'Disk I/O' | translate }}</h4>
```

Comment:
let's add a TODO here.
_Disk I/O shows network data, it will be replaced later._